network/p2p/dnsaddr/resolve.go

```diff
@@ -35,8 +35,14 @@ func Iterate(initial multiaddr.Multiaddr, controller ResolveController, f func(d
 	if resolver == nil {
 		return errors.New("passed controller has no resolvers Iterate")
 	}
+	const maxHops = 100
```

Comment:
Why 100 - i.e. this feels more like 10 is sufficient..?

Reply:
I do not know what is actual max limit